.github/workflows/linters.yml

```diff
@@ -54,3 +54,5 @@ jobs:
       - name: Check node_modules existence
         run: |
           if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
+
```

Comment:
-[ ] [OPTIONAL] Please solve your linter error. 
`Hint` - I don't think there is anything wrong with your `linters`, but the `linters` are showing an error in `node_modules`, so try reinstalling the `linters` or ignoring `node_modules` again.

Reply:
I have already fixed the linter issues. In a few moments, I will send it to review.

Reply:
Good afternoon! I have already fixed the comments on the project and I managed to solve the problem with the liners.

I will continue with the next phase. Beforehand thank you very much.